Casks/tencent-meeting.rb

```diff
@@ -18,16 +18,32 @@
   homepage "https://meeting.tencent.com/"
 
   livecheck do
-    skip "No version information available"
+    url %Q(https://meeting.tencent.com/web-service/query-download-info?q=[{"package-type":"app","channel":"0300000000","platform":"mac","arch":"#{arch}"}]&nonce=0000000000000000)
+    regex(%r{/cos/([0-9a-z]{32})/})
+    strategy :json do |json, regex|
+      info = json["info-list"][0]
+      "#{info["version"]},#{info["url"][regex, 1]}"
+    end
```

Comment:
```suggestion
    regex(%r{/cos/(\h+)/TencentMeeting[._-].+?v?(\d+(?:\.\d+)+)})
    strategy :json do |json, regex|
      json["info-list"]&.map do |item|
        match = item["url"]&.match(regex)
        next if match.blank?

        "#{match[2]},#{match[1]}"
      end
    end
```

Similar idea but this will also handle the situations where the `info-list` array is empty, contains multiple entries, or is missing entirely. This also extracts all the version information from the filename in the URL, since the cask's `version` information is interpolated into the same URL (and this approach also leads to a more common pattern for cask `livecheck` blocks).

Reply:
Thank you, @samford!